frontend/src/metabase/query_builder/components/AlertModals/CreateAlertModalContent.jsx

```diff
@@ -63,7 +63,9 @@ class CreateAlertModalContentInner extends Component {
     this.props.fetchPulseFormInput();
   }
 
-  onAlertChange = alert => this.setState({ alert });
+  onAlertChange = alert => {
+    this.setState({ alert });
+  };
```

Comment:
?

Reply:
I modified it to throw a console log in there haha